### DISCUSSION TITLE
How should I create/install an scoop package?

### DISCUSSION THREAD

**Discussion by ghost** [ScoopInstaller/Scoop#6004] [Help and Questions] [answered]

**Comment by HUMORCE** [comment, marked as answer]
You can refer [this manifest](https://github.com/ScoopInstaller/Extras/blob/master/bucket/dnspyex.json) and [App Manifests reference](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifests) to help you creates manifest for Notes app quickly. At the same time, it is recommended that you submit the manifest to [Extras bucket](https://github.com/ScoopInstaller/Extras) instead self-managed.
